README.md

```diff
@@ -1,8 +1,8 @@
 # Kasa
-English to Twi translation library/system being put together by the Ghana NLP team.
+A python library/system to interact with the African language translation, STT (Speech to Text) and TTS (Text to Speech) models put together by the Ghana NLP team.
 
 # Quickstart
-Install Kasa directly from online link:
+To use this, install Kasa directly from online link:
```

Comment:
Since this is on pypi, we should be able to install with pip as well as from source. these options should be included

Reply:
Alright so we should make reference to:
`pip install Kasa`
...rather than calling the github repository